src/main/io/serial_cli.c

```diff
@@ -739,6 +741,46 @@ static void cliCMix(char *cmdline)
 #endif
 }
 
+static void cliRxCalibration(char *cmdline)
+{
+    int i, check = 0;
+    char *ptr;
+
+    if (isEmpty(cmdline)) {
+        for (i = 0; i < MAX_CALIBRATED_RX_CHANNELS; i++) {
+            printf("rccal %u %u %u\r\n", i, masterConfig.rxConfig.calibration[i].minrc, masterConfig.rxConfig.calibration[i].maxrc);
+        }
+    } else {
+        ptr = cmdline;
+        i = atoi(ptr);
+        if (i >= 0 && i < MAX_CALIBRATED_RX_CHANNELS) {
+            int minrc, maxrc;
+            ptr = strchr(ptr, ' ');
+            if (ptr) {
+                minrc = atoi(++ptr);
+                check++;
+            }
+            ptr = strchr(ptr, ' ');
+            if (ptr) {
+                maxrc = atoi(++ptr);
+                check++;
+            }
+            if (check != 2) {
```

Comment:
`sizeof(ptr) != 2` will always evaluate to TRUE since `sizeof()` operator returns size of a given data type in bytes. It has nothing to do with length of an array. Moreover, I don't need a string length in this case. `check` variable counts number of tokens separated by spaces, there is no way to count them without parsing a string.